src/main/java/mpern/sap/commerce/build/util/SparseBootstrap.java

```diff
@@ -0,0 +1,35 @@
+package mpern.sap.commerce.build.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Extension configuration element for HybrisPluginExtension.
+ */
+public class SparseBootstrap {
+
+    private boolean enabled;
+
+    private List<String> alwaysIncluded;
+
+    public SparseBootstrap() {
+        enabled = false;
+        alwaysIncluded = new ArrayList<>();
+    }
+
+    public boolean getEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(boolean enabled) {
+        this.enabled = enabled;
+    }
+
+    public List<String> getAlwaysIncluded() {
+        return alwaysIncluded;
+    }
+
+    public void setAlwaysIncluded(List<String> alwaysIncluded) {
```

Comment:
A setter accepting a plain Collection would also be nice. Especially when using a kts config

Reply:
This is a good idea, I noticed when using it with a kts build gradle that I had to set a list object. Changed.

Reply:
I tried to have a setter with Collection but the configuration in Kotlin does not seem to like it, I always get an incompatible type error that it expects a mutable set as the getter specifies.

Suggestions?

Reply:
After sleeping a night on it: 

SparseBootstrap should use the "lazy configuration" concept of Gradle like the rest of the config. This also care of the getter/setter issues.

https://discuss.gradle.org/t/extension-with-a-nested-lazy-property/41380

Reply:
P.S.if it makes our lives easier, we can drop support for older gradle versions while we are at it. 5.x is quite old

Reply:
I managed to make it work for versions from 5.6.4 up, but we have to drop everything below it (5.5 and down). As a bonus, I fixed the issues on gradle 8 and now we can support it.